cmd/openshift-install/migrate.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"github.com/spf13/cobra"
+
+	azmigrate "github.com/openshift/installer/cmd/openshift-install/migrate/azure"
+)
+
+func newMigrateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "migrate",
+		Short: "Do a migration",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return cmd.Help()
+		},
+	}
+	cmd.AddCommand(newMigrateAzureCmd())
+	return cmd
+}
+
+func newMigrateAzureCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "azure",
+		Short: "Do a Azure migration",
+		Args:  cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return cmd.Help()
+		},
+	}
+
+	cmd.AddCommand(newMigrateAzurePrivateDNSCmd())
+	return cmd
+}
+
+func newMigrateAzurePrivateDNSCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "privatedns",
+		Short: "Do a Azure private DNS migration",
+		Args:  cobra.ExactArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return cmd.Help()
+		},
+	}
+
+	for _, subCmd := range []*cobra.Command{
+		azmigrate.NewMigrateAzurePrivateDNSEligibleCmd(),
+		azmigrate.NewMigrateAzurePrivateDNSMigrateCmd(),
+	} {
+		cmd.AddCommand(subCmd)
+	}
+
+	return cmd
+}
```

Comment:
Can we move these to the corresponding packages.

Reply:
They've been moved.